src/broadcast/broadcastmanager.cpp

```diff
@@ -92,8 +92,7 @@ void BroadcastManager::slotControlEnabled(double v) {
         // Wrap around manually .
         // Wrapping around in WPushbutton does not work
         // since the status button has 4 states, but this CO is bool
-        m_pBroadcastEnabled->set(0.0);
-        emit broadcastEnabled(false);
+        v = 0.0;
```

Comment:
The signal is still emitted, but now `m_pBroadcastEnabled` is not set in this cases, see the `else` arm. Was this intended?

Reply:
indeed :facepalm: 

let's set it in that else branch where the status is reset.
now the status in the preferences is also reset correctly.